src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -84,7 +84,13 @@ public enum BdecVersion {
     ONE(1),
 
     /** Uses Arrow to generate BDEC chunks with {@link ArrowBatchWriteMode#FILE}. */
-    TWO(2);
+    TWO(2),
+
+    /**
+     * Uses Parquet to generate BDEC chunks with {@link
+     * net.snowflake.ingest.streaming.internal.ParquetRowBuffer}.
```

Comment:
Do we plan on extending this version to support encryption or is that going to be `FOUR`? If it is not finalized we should probably document that this is a work in progress, experimental version.

Reply:
the encryption will be handled in a separate PR.
indeed version 3 is experimental for now, I will add comment.